Source/Logger/trace.cpp

```diff
@@ -43,13 +43,29 @@ int sprintf_s(char* buffer, size_t size, _Printf_format_string_ char const* form
 
 #if !HC_PLATFORM_IS_PLAYSTATION
 template<size_t SIZE>
-int vsprintf_s(char(&buffer)[SIZE], _Printf_format_string_ char const* format, va_list varArgs) noexcept
+int vstprintf_s(char(&buffer)[SIZE], _Printf_format_string_ char const* format, va_list varArgs) noexcept
 {
     return vsnprintf(buffer, SIZE, format, varArgs);
 }
 #endif
 #endif
 
+#if HC_PLATFORM_IS_MICROSOFT || HC_PLATFORM_IS_PLAYSTATION
```

Comment:
why the `|| HC_PLATFORM_IS_PLAYSTATION`? I'm pretty sure you'd end up with a broken definition

Reply:
and if you don't need the playstation bit, I'd move this to be in an if/else with the block above

Reply:
In the current version that redefines vsprintf_s, it only redefines that when not Microsoft and not Playstation (the same logic above). So this new redefinition needs to apply for both of those. Unless you're suggesting we should keep vsprintf_s for Playstation.

Reply:
Honestly we should probably just get @nassosterz-ms to weigh in here since he's actually in the ps stuff

Reply:
oh right, I misread the if above

Reply:
Changed according to chat with Nassos